homestar-runtime/src/event_handler/swarm_event.rs

```diff
@@ -555,6 +569,10 @@ async fn handle_swarm_event<THandlerErr: fmt::Debug + Send, DB: Database>(
                     );
                     if mdns.has_node(&peer_id) {
                         behaviour.kademlia.remove_address(&peer_id, &multiaddr);
+                        debug!(
+                            peer_id = peer_id.to_string(),
+                            "removed peer from kademlia table"
```

Comment:
for the log, should we call out any difference here between remove_address and remove_peer?

Reply:
Indeed. Updated it to read "removed peer address from kademlia table".